docs/data-sources/errors_application.md

```diff
@@ -0,0 +1,35 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "logtail_errors_application Data Source - terraform-provider-logtail"
```

Comment:
```suggestion
page_title: "errors_application Data Source - terraform-provider-logtail"
```

I would lose the logtail prefix please. 🙏 

The fact that telemetry and errors share a codebase is implementation detail.
We also treat errors as separate in the API as well:

<img width="620" height="186" alt="CleanShot 2025-11-26 at 15 40 25" src="https://github.com/user-attachments/assets/a9fe4763-9ea1-4124-a167-2f96517a9567" />

Reply:
For all errors and warehouse resources please 🙏

Reply:
Oh, didn't realize, okay, let's keep the prefix then